src/features/web-compat.js

```diff
@@ -531,6 +535,22 @@ export class WebCompat extends ContentFeature {
         }
     }
 
+    /**
+     * Support for modifying localStorage entries
+     */
+    modifyLocalStorage () {
+        /** @type {import('../types//webcompat-settings').WebCompatSettings['modifyLocalStorage']} */
```

Comment:
Make sure the type change is within webcompat-settings.d.ts, that will fix the unit test fail.

Reply:
^ yeah as @jonathanKingston mentioned, the `webcompat-settings.d.ts` needs updating based on the schema. The change sequence should be something like

1: create your change in the privacy config repo (eg: updating the schema)
2: either `npm link` locally or install the branch locally into CSS
3: run `npm run build-types` <- this will ensure your PR has the types you added to the webCompat schema. 

Ping me if there are any issues, and I'll go over docs/guides this week to ensure we make this flow clearer/easier :)

Reply:
Thanks! I was struggling a bit with that. I ended up pushing the package.json change that points at [my privacy config branch](https://github.com/duckduckgo/privacy-configuration/pull/2283) to get CI to pass, let's be sure to revert that after that PR is merged.